client/knowledge_base_search/src/containers/SearchContainer/SearchContainer.js

```diff
@@ -43,62 +51,54 @@ class SearchContainer extends Component {
   render() {
     return (
       <section className="_full-width-row search_container--section">
-        <div className="_container">
-          <CSSTransitionGroup
-            transitionName='search_header'
-            transitionEnterTimeout={500}
-            transitionLeaveTimeout={300}
-          >
-            {!this.props.hasResults
-              ? (
-                <h2 className="header--input" key='header_input'>
-                  Use a community's expertise to amplify the way you find information.
-                </h2>
-                )
-              : null
-            }
-          </CSSTransitionGroup>
+        <div className="_container _container_large">
           <form onSubmit={this.handleOnSubmit}>
-            <div className="positioned--icon">
-              <Icon type="search" />
-            </div>
-            <TextInput
-              id="search_input"
-              placeholder="Enter words, phrase, or a question about travel"
-              value={this.state.search_input}
-              onInput={this.handleOnInput}
-              style={{width: '100%'}}
-              disabled={this.props.isFetching}
-            />
-            <div className="_container-right">
-              <button
-                type="button"
-                className="random-query--button"
-                onClick={this.handleOnClick}
-                disabled={this.props.isFetching}
-              >
-                Random Query
-              </button>
-            </div>
-            <CSSTransitionGroup
-              transitionName='search_button'
-              transitionEnterTimeout={500}
-              transitionLeaveTimeout={300}
-            >
-              {!this.props.hasResults
-                ? (
-                  <div className="_container-center">
-                    <button
-                      className="white--button"
-                      disabled={this.props.isFetching}
-                    >
-                      Retrieve Answers
-                    </button>
-                  </div>
-                  )
-                : null
-              }
-            </CSSTransitionGroup>
+            <Tabs selected={0}>
+              <Pane label="Preset questions">
+              <div className='clickable-tabs'>
+                {
+                  this.state.random_queries.map((query, index) => {
+                    return (
+                      <span key={'questionButton-'+index}>
+                        <button className={query === this.state.search_input
```

Comment:
probably should provide `disabled={this.props.isFetching}` so users can't spam the system